### ISSUE TITLE
Preprocessing the test set and the ID scores

### ISSUE THREAD

**Issue by three-bee** [anantarb/triplanenet#3]
Hello, thanks for the great work! We have several questions:

The paper states to have used 2824 CelebA-HQ images as the test set. However, 4 of them could not pass the pre-processing step; hence, 2820 of them were utilized while calculating metrics. We were trying to replicate the results for our data, but unfortunately, the pre-processing scripts you provided yielded 2808 in our case, not 2820. We were wondering if there is a problem with the pre-processing scripts. Maybe you could share the idx's of 2820 images you have utilized so we can double-check our dataset.

As a further result, we achieved an ID score of 0.79 with our data. Table 2 states that it should be 0.88, so we were wondering if it is a typo or due to the dataset. 

Thanks in advance

**Comment by anantarb** [comment]
Hi, 
Regarding the pre-processing script there is a problem with the pre-processing script. We adapted the pre-processing script from EG3D but later found out that there was a problem. If you look at lines 27 and 28 in dataset_preprocessing/ffhq /crop_images_in_the_wild.py, the code sorts both image files and landmark files. However, if the face cannot be detected in some images and no landmark file is present, then this code will yield incorrect cropping of the images. Therefore, I would recommend fixing this before running the evaluation. Unfortunately, we found out this after the paper release. If you still want those incorrectly cropped images we used in our evaluation, I can share them via google drive. 

Regarding the ID score, 0.79 seems correct, as we have specified in the "Major Changes" section in the readme.md that we drop normalization in the code. With normalization, 0.79 is equivalent to 0.89, which is in a similar range as reported in the paper.

Let me know if you have any further questions.
